gigalixir/netrc.py

```diff
@@ -1,41 +1,42 @@
 from __future__ import absolute_import
 import netrc
 import os
+import platform
 
-def clear_netrc():
+def netrc_name():
+    if platform.system().lower() == 'windows':
```

Comment:
I think this is fine for now since the complexity is low, but I prefer not to check the platform "again" here. Ideally, the only platform check would be done in __init__.py's cli function. For example, following the pattern used by the "opener" or "router". I can do a follow-up change to refactor this unless you want to take a stab at it.

Reply:
Yeah, I agree. I was originally trying to store it in the ctx.obj dictionary, but I couldn't see a good way of then getting that down into the functions that needed it.
I suppose since the only info is a string, I could store it in a environment variable for the duration of the script. Your thoughts?

Reply:
I think it's fine as-is. It's something that should be easy to refactor later on. Thanks again.